parm/chem/ExtData.other

```diff
@@ -11,12 +11,12 @@ TROPP            'Pa'    Y       N                  -            0.0      1.0
 DU_SRC            NA  N Y - none none du_src     ExtData/Dust/gocart.dust_source.v5a.x1152_y721.nc
 
 # FENGSHA input files. Note: regridding should be N or E
-DU_CLAY          '1'  Y E           -          none none clayfrac   ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
-DU_SAND          '1'  Y E           -          none none sandfrac   ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
-DU_SILT          '1'  Y E           -          none none siltfrac   /dev/null
-DU_SSM           '1'  Y E %y4-%m2-%d2T12:00:00 none none ssm        ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
-DU_RDRAG         '1'  Y E           -          none none drag_part  ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
-DU_UTHRES        '1'  Y E           -          none none uthres     ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
+DU_CLAY           '1'  Y E           -           none none clayfrac    ExtData/dust/FENGSHA_SOILGRIDS2019_GEFSv12_v1.2.nc
```

Comment:
This path changed the uppercase 'D' in Dust into lowercase, but the directory names didn't actually change.

Reply:
You are right the filenames should have changed here.  

They should be 

```bash
DU_CLAY           '1'  Y E           -           none none clayfrac    ExtData/Dust/FENGSHA_SOILGRIDS2019_GEFSv12_v1.2.nc
DU_SAND           '1'  Y E           -           none none sandfrac    ExtData/Dust/FENGSHA_SOILGRIDS2019_GEFSv12_v1.2.nc
DU_SILT           '1'  Y E           -           none none siltfrac    /dev/null
DU_SSM            '1'  Y E           -           none none ssm         /dev/null:1.0
DU_RDRAG          '1'  Y E %y4-%m2-%d2t12:00:00  none none albedo_drag ExtData/Dust/FENGSHA_Albedo_drag_v1.nc
DU_UTHRES         '1'  Y E           -           none none uthres      ExtData/Dust/randomforestensemble_uthres.nc
```

And the path that you changed the ExtData to point to is not synced with that path that was there. I don't understand why this was modified when it was fine with pointing to Raffaele's directory.  It should continue to point to a location that the aerosol team has access to. 


You changed it from here `/scratch1/RDARCH/rda-arl-gpu/Barry.Baker/emissions/NASA` to `/scratch2/NCEPDEV/ensemble/noscrub/Walter.Kolczynski/global-workflow/gocart_emissions/Dust/`